src/video.tsx

```diff
@@ -17,11 +17,17 @@ export const RATIO = {
     '4:3': 4 / 3,
 };
 
-type VideoSource = {
+type VideoSourceWithType = {
     src: string;
     type?: string; // video/webm, video/mp4...
```

Comment:
I'd call it `mimeType` in order to convey full semantics of the prop to future developers :)